openstack/hermes/templates/api-deployment.yaml

```diff
@@ -19,55 +19,60 @@ spec:
     spec:
       nodeSelector:
         zone: farm
-      volumes:
-        - name: config
-          configMap:
-            name: hermes
+
       containers:
-        - name: api
-          image: {{required ".Values.hermes.image is missing" .Values.hermes.image}}:{{.Values.hermes.image_tag | replace "string:" ""}}
-          imagePullPolicy: {{required ".Values.hermes.image_pull_policy is missing" .Values.hermes.image_pull_policy}}
-          args:
-            - -f
-            - /etc/hermes/hermes.conf
-          env:
-            - name: HERMES_DEBUG
-              value: '{{.Values.hermes.debug}}'
-            - name: HERMES_INSECURE
-              value: '{{.Values.hermes.insecure}}' # SSL certificates are hard :(
-            - name: HERMES_ES_USERNAME
-              valueFrom:
-                secretKeyRef:
-                  name: hermes-es-api
-                  key: HERMES_ES_USERNAME
-            - name: HERMES_ES_PASSWORD
-              valueFrom:
-                secretKeyRef:
-                  name: hermes-es-api
-                  key: HERMES_ES_PASSWORD
-          volumeMounts:
-            - mountPath: /etc/hermes
-              name: config
-          resources:
-            requests:
-              memory: {{required ".Values.resources_requests_memory_api is missing" .Values.resources_requests_memory_api}}
-              cpu: {{required ".Values.resources_requests_memory_api is missing" .Values.resources_requests_cpu_api}}
-          # livenessProbe:
-          #   #exec: 
-          #   #  command:
-          #   #    - /bin/false
-          #   #  initialDelaySeconds: 21600
-          #   #  periodSeconds: 21600 # restart api every 6 hours
-          #   httpGet:
-          #     path: /v1/
-          #     port: 80
-          #   timeoutSeconds: 10
-          #   periodSeconds: 60
-          #   initialDelaySeconds: 60
-          #   readinessProbe:
-          #     httpGet:
-          #       path: /v1/
-          #       port: 80
-          #     timeoutSeconds: 5
-          #     periodSeconds: 5
+      - name: api
+        image: {{required ".Values.hermes.image is missing" .Values.hermes.image}}:{{.Values.hermes.image_tag | replace "string:" ""}}
```

Comment:
Is the `replace "string:" ""` still required? Checking our internal values I was not able to find a tag with this prefix.